### ISSUE TITLE
Fix `renderExpando` issue

### ISSUE THREAD

**Issue by gaurav21r** [super-elements/super-grid#2]
@lakshay1994 Good work on https://github.com/SitePen/dgrid/issues/1244!

Lets make relevant changes to `super-grid` as well.

**Comment by lakshay1994** [comment]
**super-grid** with the **Tree** attribute is expanding and collapsing properly only on the data already stored in the **store**  but if we add more data in the store(like through console), that data is not structured like a tree.

**Comment by gaurav21r** [comment]
@lakshay1994 Ok, as you mentioned, we need to call `getRootCollection()` when changing the `value` of `super-grid` right?

**Comment by lakshay1994** [comment]
Yes

**Comment by gaurav21r** [comment]
We are using **JS** Getters and Setters for `value` property and dynamically changing the `collection` property of dgrid on every `set()`.

Try changing https://github.com/super-elements/super-grid/blob/master/super-grid.js#L258 to `this.dgrid.set('collection', this.store.getRootCollection());

**Comment by lakshay1994** [comment]
Yes, thank you. now the data is well presented in the tree form by calling the **getRootCollection()**  on the **store** after new data is entered to the super-grid.
